WPFHospitalEditor/Controller/Interface/IMapObjectController.cs

```diff
@@ -9,5 +9,6 @@ interface IMapObjectController
         MapObject update(MapObject mapObject);
         List<MapObject> getOutterMapObjects();
         MapObject findMapObjectById(int id);
+        void checkMapObjectSearchInput(string textBoxInput, string comboBoxTextInput);
```

Comment:
for all data contained in the class, as well as for methods, you should write their accessibility (private, public, protected,..)